pkg/domain/domain.go

```diff
@@ -1944,6 +1944,35 @@ func decodePrivilegeEvent(resp clientv3.WatchResponse) PrivilegeEvent {
 	return msg
 }
 
+func batchReadMoreData(ch clientv3.WatchChan, event PrivilegeEvent) PrivilegeEvent {
+	timer := time.NewTimer(5 * time.Millisecond)
+	defer timer.Stop()
+	const maxBatchSize = 128
+	for i := 0; i < maxBatchSize; i++ {
+		select {
+		case resp, ok := <-ch:
+			if !ok {
+				return event
```

Comment:
Is it ok to rerun the event here？
The later use may be wrong？err := privReloadEvent(do.privHandle, &event)

Reply:
This function just focus on the batching functionality, and the first input event  is a correct one passed by the caller.
The close of the channel will be handled by the reloadPrivilegeLoad instaed of this function.